webcompat/webhooks/helpers.py

```diff
@@ -97,24 +97,6 @@ def extract_priority_label(body):
     return None
 
 
-def update_issue(payload, issue_number):
```

Comment:
Presumably we're putting this back into `new_opened_issue` so it's easier to test?